packages/ra-ui-materialui/src/layout/UserMenu.tsx

```diff
@@ -37,7 +37,7 @@ const TransformOrigin: PopoverOrigin = {
     horizontal: 'right',
 };
 
-const UserMenu = props => {
+const UserMenu: FC<UserMenuProps> = props => {
```

Comment:
Please avoid `FC` and type the props directly. See https://github.com/facebook/create-react-app/pull/8177. We're trying to avoid it for new code and migrate old code  progressively.